csatar-plugins/csatar/components/recordlist/tableRows.htm

```diff
@@ -0,0 +1,28 @@
+{% set records = __SELF__.records %}
+{% set noRecordsMessage = __SELF__.noRecordsMessage %}
+
+{% for record in records %}
+{% if detailsPage %}<a href="{{ detailsPage|page({ (detailsUrlParameter): attribute(record, detailsKeyColumn) }) }}">{% endif %}
+    {% partial __SELF__~'::tableRow' record = record %}
+    {% if detailsPage %}</a>{% endif %}
+{% else %}
+<p>{{ noRecordsMessage }}</p>
+{% endfor %}
```

Comment:
Nem szukseges egy "endif" ez az "endfor" ele?

Reply:
nem, itt az else a for loopra vontakozik, az if az 5-ös és 7-e sorban be van zárva a sor végén